lib/factory_bot/syntax/methods.rb

```diff
@@ -51,22 +51,51 @@ module Methods
       # Generates a hash of attributes for a registered factory by name.
       # @return [Hash] hash of attributes for the factory
 
-      # @!method build_list(name, amount, *traits_and_overrides)
+      # @!method null(name, *traits_and_overrides, &block)
```

Comment:
Is there a use case for using the `null` strategy directly? It always returns `nil`. I am tempted to leave out the documentation for `null`.

Reply:
Good call, I actually didn't realise it was just for internal purposes. I can't think of a purpose either so I'll remove the documentation for `null`.